ksql-cli/src/main/java/io/confluent/ksql/cli/Cli.java

```diff
@@ -274,7 +277,7 @@ private String readLine() throws IOException {
         if (result == null) {
           throw new EndOfFileException();
         } else {
-          return result.trim();
+          return result.trim().replace((char) 0xa0, ' ');
```

Comment:
Is this needed if `handleLine` is also replacing chars? Would be good to have this whitespace handling in a single place if we can.